zapisy/apps/schedule/urls.py

```diff
@@ -27,4 +27,6 @@
     path('session/', views.session, name='session'),
     path('session/feed/', feeds.LatestExams(), name='session_feed'),
     path('events/report/', views.events_report, name='events_report'),
+    path('freeday/', views.FreedayAPIView.as_view(), name='freeday'),
+    path('changeday/', views.ChangedDayAPIView.as_view(), name='changeday'),
```

Comment:
Czy to, że tu używamy liczby pojedynczej, wynika z jakiejś REST-owej konwencji? Bo jeśli nie, to wolałbym liczbę mnogą (zarówno w URI jak i nazwie). A na pewno zmieniłbym
```suggestion
    path('freeday/', views.FreedayAPIView.as_view(), name='freeday'),
    path('changedday/', views.ChangedDayAPIView.as_view(), name='changedday'),
```
(i oczywiście inne wystąpienia)